imagepicker/src/main/java/com/esafirm/imagepicker/view/SnackBarView.kt

```diff
@@ -0,0 +1,80 @@
+package com.esafirm.imagepicker.view
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import android.view.animation.Interpolator
+import android.widget.Button
+import android.widget.RelativeLayout
+import android.widget.TextView
+import androidx.annotation.StringRes
+import androidx.core.view.ViewCompat
+import androidx.interpolator.view.animation.FastOutLinearInInterpolator
+import com.esafirm.imagepicker.R
+import kotlinx.android.synthetic.main.ef_imagepikcer_snackbar.view.*
+
+class SnackBarView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyle: Int = 0
+) : RelativeLayout(context, attrs, defStyle) {
+
+    private var txtCaption: TextView? = null
+    private var btnAction: Button? = null
+
+    init {
+        inflate(context, R.layout.ef_imagepikcer_snackbar, this)
+        val padding = context.resources.getDimensionPixelSize(R.dimen.ef_spacing_double)
+        setPadding(padding, 0, padding, 0)
+
+        if (!isInEditMode) {
+            val height = context.resources.getDimensionPixelSize(R.dimen.ef_height_snackbar)
+            translationY = height.toFloat()
+            alpha = 0f
+            txtCaption = ef_snackbar_txt_bottom_caption
+            btnAction = ef_snackbar_btn_action
+        }
+    }
+
+    fun setText(@StringRes textResId: Int) {
+        txtCaption?.setText(textResId)
+    }
+
+    fun setOnActionClickListener(@StringRes textId: Int, onClickListener: OnClickListener) {
+        val mTextId = if (textId == 0) R.string.ef_ok else textId
+        btnAction?.let {
+            setText(mTextId)
+            setOnClickListener { v: View? -> hide { onClickListener.onClick(v) } }
+        }
+    }
+
+    @SuppressLint("Recycle")
```

Comment:
Can you elaborate on this lint? Maybe we're doing something bad 😅

Reply:
honestly, I don't know what this does, the IDE recommend this line. Been looking around but can't find a explanation. Sorry will remove this line.